tokio/src/runtime/basic_scheduler.rs

```diff
@@ -238,6 +248,8 @@ impl<P: Park> Inner<P> {
                         Some(entry) => entry,
                         None => {
                             // Park until the thread is signaled
+                            scheduler.metrics.incr_park_count();
+                            scheduler.metrics.submit(&scheduler.spawner.shared.metrics);
```

Comment:
What if we busy loop because a task is always ready. Metrics should still get updated. 

Is this handled by line 271 which will also yield after some events had been processed?

Reply:
Yes, line 271 ensures that metrics are submitted either way.